src/core/server/integration_tests/saved_objects/migrations/zdt_1/basic_downgrade.test.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Path from 'path';
+import fs from 'fs/promises';
+import { range } from 'lodash';
+import { createTestServers, type TestElasticsearchUtils } from '@kbn/core-test-helpers-kbn-server';
+import { SavedObjectsBulkCreateObject } from '@kbn/core-saved-objects-api-server';
+import '../jest_matchers';
+import { getKibanaMigratorTestKit } from '../kibana_migrator_test_kit';
+import { delay, parseLogFile, createType } from '../test_utils';
+import { getBaseMigratorParams } from '../fixtures/zdt_base.fixtures';
+
+export const logFilePath = Path.join(__dirname, 'basic_downgrade.test.log');
+
+describe('ZDT upgrades - basic downgrade', () => {
+  let esServer: TestElasticsearchUtils['es'];
+
+  const startElasticsearch = async () => {
```

Comment:
NIT Since this function is very common, I made it available directly in the `kibana_migrator_test_kit.ts` (under the same name).

Reply:
Oh, great. Wasn't aware of that and TBH I thought about extracting it in this PR. Will adapt accordingly